packages/gestalt-design-tokens/tokens/vr-theme/base/lineheight.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
can we add this under font? lineheight is actually a font token it's not a category by itself.

Reply:
We will do this in the July 2024 release, for now this will keep things 1:1 with our existing repo